conductr_cli/conduct_load.py

```diff
@@ -167,6 +189,10 @@ def load_v2(args):
 
     initial_bundle_file_name, bundle_file = resolver.resolve_bundle(custom_settings, bundle_resolve_cache_dir,
                                                                     args.bundle, args.offline_mode)
+
+    if not is_bundle(bundle_file):
+        bundle_file = invoke_bndl(bundle_file)
+
```

Comment:
This is the bulk of it. If we didn't resolve a bundle, invoke `bndl` and get the path to a bundle that it creates.